addons/stock/migrations/8.0.1.1/post-migration.py

```diff
@@ -530,12 +531,19 @@ def get_location_by_usage(usage):
 def migrate_stock_warehouses(cr, registry):
     """Migrate all the warehouses"""
     warehouse_obj = registry['stock.warehouse']
+    # Add a code to all warehouses that have no code
+    openupgrade.logged_query(
+        cr, """
+        UPDATE stock_warehouse SET code= 'WH' || to_char(id, 'FM999MI')
```

Comment:
@StefanRijnhart 
Right, id field is used to generate unique code per warehouse
In the stock addons, i see that the code field is required, i think after this update we must alter the field to add NOT NULL like
ALTER TABLE stock_warehouse ALTER COLUMN code SET NOT NULL; 

regards,